StripePaymentSheet/StripePaymentSheet/Source/Internal/API Bindings/v1-elements-sessions/ExternalPaymentMethod.swift

```diff
@@ -6,21 +6,35 @@
 //
 
 import Foundation
+@_spi(STP) import StripeCore
 @_spi(STP) import StripePayments
 
-/// ViewModel-like information for displaying external payment methods (EPMs)
+/// ViewModel-like information for displaying external payment methods (EPMs), delivered in the `v1/elements/sessions` response.
+/// - Seealso: https://git.corp.stripe.com/stripe-internal/pay-server/blob/master/lib/elements/api/private/struct/external_payment_method_data.rb
 struct ExternalPaymentMethod: Decodable, Equatable, Hashable {
     /// The type of the external payment method. e.g. `"external_foopay"`
     /// These match the strings specified by the merchant in `ExternalPaymentMethodConfiguration`.
     let type: String
     /// A localized label for the payment method e.g. "FooPay"
-    let localizedLabel: String
+    let label: String
+    /// URL of a 48x pixel tall PNG representing the payment method suitable for display against a light background color.
+    let lightImageUrl: URL
+    /// URL of a 48x pixel tall PNG representing the payment method suitable for display against a dark background color. If `nil`, use `lightImageUrl` instead.
+    let darkImageUrl: URL?
```

Comment:
Is there any guidance on width of the PNG we would like provided? Should it be 48x48?